code/API_definitions/CAMARA/number_verification.yaml

```
@@ -132,11 +134,64 @@ paths:
       security:
         - three_legged:
           - number-verification-share-read
+  /verify-ip:     
+    post:
+      tags:
+        - IP phone number verify
+      summary: Validates that the phone number of the mobile device connected via the cellular network is using the IP address of the connection to the server of the service that is querying the API.
+      description: |-
+        The Mobile User inform phone number to verification.
+         - BackEnd APP or Web Partner Application Server pass the IP address to MNO API GTW to challenge and validate phone number with IP matched.
+         - It returns true/false depending on if the phone number and IP address received as input match the device IP address associated to the mobile network operator data connection.
+      operationId: ipPhoneNumberVerify
+      parameters:
+        - in: header
+          name: x-correlator
+          required: false
+          description: Correlation id for the different services
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+               $ref: '#/components/schemas/ueId'
+        required: true  
+      responses:
+        '200':
+          description: OK
+          headers:
+            x-correlator:
+              description: Correlation id for the different services
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NumberVerificationMatchResponse'
+        '400':
+          $ref: '#/components/responses/InvalidArgument'
+        '500':
+          $ref: '#/components/responses/Internal'
+        '503':
+          $ref: '#/components/responses/Unavailable'
+        '504':
+          $ref: '#/components/responses/Timeout'
+      security:
+        - oAuth2ClientCredentials:
+          - verify-ip        
 components:
   securitySchemes:
     three_legged:
       type: openIdConnect
       openIdConnectUrl: https://example.com/.well-known/openid-configuration
+    oAuth2ClientCredentials:
+          type: oauth2
+          description: Client Credentials – Intended for the server-to-server authentication between service provider and MNO.
```

```suggestion
          description: Client Credentials - Intended for the server-to-server authentication between service provider and MNO.
```
I think here the sign that was used is not a dash.